packages/components/src/List/Toolbar/ColumnChooserButton/ColumnChooser/ColumnChooser.scss

```diff
@@ -0,0 +1,84 @@
+$tc-popover-border-color: rgb(226, 226, 226) !default;
+$tc-popover-border-width: 0.1rem !default;
+$tc-border-color: #ededed !default;
+$tc-icon-size: 1.2rem !default;
+$tc-height: 4rem;
+
+.tc-column-chooser {
+	:global(.tc-tooltip-body) {
+		margin: 0;
+	}
+
+	:global(.tc-tooltip-footer.tc-column-chooser-footer) {
+		justify-content: flex-end;
+	}
+
+	&-aria-hidden {
+		position: absolute;
+		left: -200em;
+		top: -20em;
+	}
+
+	&-header {
+		display: flex;
+		align-items: center;
+		&-title {
+			font-weight: bold;
+		}
+	}
+
+	&-filter {
+		margin: 0 $padding-small 0 $padding-small;
+	}
+
+	&-body {
+		overflow-y: auto;
+		width: 100%;
+	}
+
+	&-footer {
+		align-items: center;
+		display: flex;
+	}
+
+	&-row {
+		padding: 0 $padding-large 0 $padding-large;
+		display: flex;
+		align-items: center;
+		height: $tc-height;
+		min-height: $tc-height;
+
+		&-select-all {
+			border-bottom: $tc-popover-border-width solid $tc-popover-border-color;
+		}
+
+		&:hover {
+			background: $wild-sand;
+		}
+
+		:global(.checkbox.tc-toggle) {
+			display: flex;
+		}
+
+		&-label,
+		:global(.checkbox.tc-toggle > label span) {
+			font-size: 1.4rem;
+			padding-left: $padding-large;
+		}
+
+		:global(.checkbox.tc-toggle > label) {
+			padding-left: $padding-small;
+			overflow: hidden;
+			text-overflow: ellipsis;
+			white-space: nowrap;
+			max-width: 21rem;
+			min-height: 0;
+		}
+
+
+		svg[name='talend-locked'] {
```

Comment:
```suggestion
		svg {
```

Reply:
why isn't it better to precise the svg I want to resize ? 
What if tomorrow I have another swg under row ?